config/webpack.dev.js

```diff
@@ -23,6 +23,7 @@ const HOST = process.env.HOST || 'localhost';
 const PORT = process.env.PORT || 3000;
 const HMR = helpers.hasProcessFlag('hot');
 // if env is 'inmemory', the inmemory debug resource is used
+const OPENSHIFT_CONSOLE_URL = process.env.OPENSHIFT_CONSOLE_URL || 'https://console.starter-us-east-2.openshift.com/console/';
```

Comment:
yeah - in dev I'm using this script right now in dev:
https://github.com/fabric8io/fabric8-ui/blob/master/environments/openshift-prod-cluster.sh

ideally we'd have a pre-prod ;). Then the next best thing is having pre-prod SaaS services that all point to the prod KeyCloak and OSO so at least we've pre-prod for forge + planner + core SaaS stuff